demo/scripts/load-driver-image-into-kind.sh

```diff
@@ -27,6 +27,16 @@ set -o pipefail
 
 source "${CURRENT_DIR}/common.sh"
 
-kind load docker-image \
-	--name "${KIND_CLUSTER_NAME}" \
-	"${DRIVER_IMAGE}"
+# Work around kind not loading image with podman
+if [[ "${CONTAINER_TOOL}" == "podman" ]]; then
+	IMAGE_ARCHIVE=driver_image.tar
+	${CONTAINER_TOOL} save -o "${IMAGE_ARCHIVE}" "${DRIVER_IMAGE}" && \
+	kind load image-archive \
+		--name "${KIND_CLUSTER_NAME}" \
+		"${IMAGE_ARCHIVE}"
+	rm "${IMAGE_ARCHIVE}"
+else
+	kind load docker-image \
+		--name "${KIND_CLUSTER_NAME}" \
+		"${DRIVER_IMAGE}"
+fi
```

Comment:
Do these also need `KIND_EXPERIMENTAL_PROVIDER=${CONTAINER_TOOL} ` in front of them?

Reply:
While we're at it, let's just drop the if/else and do it the same for both podman and docker (i.e. the new podman way).

Reply:
Also, mabey the better way to do this is to introduce a `KIND` variable instead of calling `kind` directly. Then in common.sh we define `KIND="KIND_EXPERIMENTAL_PROVIDER=${CONTAINER_TOOL} kind"` and replace all direct calls to `kind` with `${KIND}`.

Reply:
> While we're at it, let's just drop the if/else and do it the same for both podman and docker (i.e. the new podman way).

I can't currently test it. See my earlier  response.

Reply:
> Do these also need `KIND_EXPERIMENTAL_PROVIDER=${CONTAINER_TOOL} ` in front of them?

According to the docs, it only works for `kind create cluster` (looking at the code seems to confirm that), will be ignored with all other commands. We still can implement your suggestion about `${KIND}` variable, hopefully this won't cause confusion.

Reply:
Ok, I've made the requested changes, also tested on Ubuntu+Docker. PTAL